src/python/BuildingControlsSimulator/DataClients/DataClient.py

```diff
@@ -307,6 +307,21 @@ def get_data(self):
             simulation_epw_dir=self.simulation_epw_dir,
         )
 
+        #add nsrdb solar data fields
+        self.weather.fill_nsrdb_data = self.weather.get_nsrdb(
```

Comment:
self.weather.get_nsrdb(sim_config=self.sim_config) can have the sim_config as the only argument.

Reply:
true but does this matter?  i prefer having the lat, long, year definition clearly laid out.

Reply:
this is more for future proofing, for example, if we add control flow for how data sources are used (e.g. if we wanted temp/rh from NSRDB to be used), it would be done within the sim_config so that it could be done programmatically.

Reply:
done.